integration_test/agents/agents.go

```diff
@@ -813,9 +872,11 @@ func SetupOpsAgentFrom(ctx context.Context, logger *log.Logger, vm *gce.VM, conf
 		if err := gce.UploadContent(ctx, logger, vm, strings.NewReader(config), util.GetConfigPath(vm.ImageSpec)); err != nil {
 			return fmt.Errorf("SetupOpsAgentFrom() failed to upload config file: %v", err)
 		}
-		if err := restartOpsAgent(ctx, logger, vm); err != nil {
+		if err := RestartOpsAgent(ctx, logger, vm); err != nil {
 			return err
 		}
+	} else if isUAPPluin {
```

Comment:
I think this is better at the top of the if clause to prevent cases where ethe config > 0 and we are using UAP

Reply:
Emm I think my logic here is right. 

regular ops agent is started once installed. However, ops agent plugin needs to be explicitly started after install.


when `len(config) >0`, both the regular and the plugin Ops Agent need a (re)start.  A `restart` is needed for the regular agent to apply the config. A `start` is needed for the plugin agent since it has never been started yet. 


When `len(config) == 0` only the plugin Ops Agent need to be started.